packages/veritone-react-common/src/components/PDFViewer/SimplePDFViewer.js

```diff
@@ -0,0 +1,199 @@
+import React, { Component, PureComponent } from 'react';
+import { string, number, func, shape, object } from 'prop-types';
+import { Document, Page, setOptions } from 'react-pdf';
+import { FixedSizeList, shouldComponentUpdate } from 'react-window';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import styles from './styles.scss';
+
+setOptions({
+  workerSrc: '//cdnjs.cloudflare.com/ajax/libs/pdf.js/2.0.305/pdf.worker.js'
+});
+const SCROLLBAR_MARGIN = 20;
+const PAGE_MARGIN = 10;
+
+class SimplePDFViewer extends PureComponent {
+  static propTypes = {
+    file: string.isRequired,
+    onItemsRendered: func,
+    userScale: number,
+    overrideScale: number,
+    onDocumentLoad: func,
+    makeTextRenderer: func,
+    initialPageOffset: number,
+    listRef: shape({ current: object }),
+    listOuterRef: shape({ current: object }),
+    searchText: string,
+    currentSearchMatch: number,
+    currentSearchPage: number
+  };
+  static defaultProps = {
+    userScale: 1,
+    overrideScale: null,
+    initialPageOffset: 1,
+    searchText: '',
+    currentSearchMatch: null
+  };
+  state = {
+    currentPageIndex: null,
+    numPages: null,
+    originalPageDimensions: null,
+    pdf: null
+  };
+
+  onDocumentLoad = pdf => {
+    this.setState({ numPages: pdf.numPages, pdf });
+    pdf.getPage(1).then(this.handlePageDimensions);
+    if (this.props.onDocumentLoad) {
+      this.props.onDocumentLoad(pdf);
+    }
+  };
+
+  handlePageDimensions = page => {
+    const scale = 1;
+    const viewport = page.getViewport(scale);
+    if (viewport) {
+      const originalPageDimensions = {
+        height: viewport.height,
+        width: viewport.width
+      };
+      this.setState({
+        originalPageDimensions
+      });
+    }
+    return page;
+  };
+
+  onItemsRendered = ({
+    overscanStartIndex,
+    overscanStopIndex,
+    visibleStartIndex,
+    visibleStopIndex
+  }) => {
+    if (this.props.onItemsRendered) {
+      this.props.onItemsRendered({
+        currentPageIndex: visibleStopIndex,
+        numPages: this.state.numPages
+      });
+    }
+  };
+
+  render() {
+    return (
+      <AutoSizer>
+        {({ width, height }) => {
+          const { numPages, originalPageDimensions } = this.state;
+          const {
+            file,
+            userScale,
+            makeTextRenderer,
+            listRef,
+            listOuterRef,
+            overrideScale,
+            initialPageOffset,
+            searchText,
+            currentSearchMatch,
+            currentSearchPage
+          } = this.props;
+          const scale =
+            overrideScale ||
+            (originalPageDimensions
+              ? (userScale * width - (SCROLLBAR_MARGIN + PAGE_MARGIN)) /
+                originalPageDimensions.width
+              : null);
```

Comment:
I feel like `scale` should probably have a fallback value here instead of `null`